### DISCUSSION TITLE
Split XCT in small and specific packages

### DISCUSSION THREAD

**Discussion by GUOLDEV** [CommunityToolkit/Maui#175] [New Feature Discussions]
I think it would work better if we could split the XCT into small and specific packages. It would make it easier to maintain, contributors could focus more on the area they have interest and the release process can be faster.

Example: 

Xamarin.CommunityToolkit.Views
Xamarin.CommunityToolkit.Converters
Xamarin.CommunityToolkit.Behaviours

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
